### PR TITLE
Help menu

### PR DESCRIPTION
**Pull request by connorreinh**
I added a help menu to the navbar which informs users of the controls for the rover. Maybe @karansingh098 can add some CSS to this at some point to make it look better.

### CODE REVIEW

**Review by evstraw — APPROVED** [review]
Looks good to me!

File: client/src/components/help-components/help-component.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+                </ul>
+                <br />
+                <h3>Arm Controls</h3>
+                <ul>
+                    <li>Q/A: arm base</li>
+                    <li>W/S: shoulder</li>
+                    <li>E/D: elbow</li>
+                    <li>R/F: forearm</li>
+                    <li>T/G/Y/H: wrist</li>
+                    <li>U/J: hand</li>
```
Comment (nhatch) [inline client/src/components/help-components/help-component.tsx:30]: The disadvantage of this (like all documentation) is that if we ever change the controls, we'll need to remember to update the documentation :upside_down_face:

**Review by nhatch — APPROVED** [review]
LGTM as well. I'll merge this

**Comment by nhatch** [comment]
Screenshot for posterity 
Screenshot from 2021-06-28 20-48-03

**Pull request merged**